apps/pie-storybook/test/utilities/index.test.ts

```diff
@@ -86,9 +86,6 @@ describe('createVariantStory', () => {
 
         const story = createVariantStory(template, propOptions, storyOpts);
 
-        // Check if the parameters include the background color
-        expect(story.parameters.backgrounds.default).toBe('background-subtle');
```

Comment:
This assertion was causing the test to fail as the `backgrounds` property is no longer set as part of `createVariantStory` This wasn't caught as the tests weren't running in CI 👍🏼